daedalus/src/Daedalus/BackendApi.purs

```diff
@@ -74,70 +74,89 @@ postRBody urlPath content = flip makeRequest urlPath $
-deleteWallet addr = postR ["delete_wallet", _address addr]
+deleteWallet addr = deleteR ["wallets", _address addr]
+
+importKey :: forall eff. String -> Aff (ajax :: AJAX | eff) CWallet
+importKey = postRBody ["wallets/keys"]
 
+restoreWallet :: forall eff. CWalletInit -> Aff (ajax :: AJAX | eff) CWallet
+restoreWallet = postRBody ["wallets/restore"]
+--------------------------------------------------------------------------------
+-- ADDRESSSES ------------------------------------------------------------------
 isValidAddress :: forall eff. CCurrency -> String -> Aff (ajax :: AJAX | eff) Boolean
-isValidAddress cCurrency addr = getR ["valid_address", showCCurrency cCurrency, addr]
+isValidAddress cCurrency addr = getR ["addresses", addr, "currencies", showCCurrency cCurrency]
+--------------------------------------------------------------------------------
+-- PROFILES --------------------------------------------------------------------
+getProfile :: forall eff. Aff (ajax :: AJAX | eff) CProfile
+getProfile = getR ["profile"]
 
-blockchainSlotDuration :: forall eff. Aff (ajax :: AJAX | eff) Int
-blockchainSlotDuration = getR ["slot_duration"]
+updateProfile :: forall eff. CProfile -> Aff (ajax :: AJAX | eff) CProfile
+updateProfile = postRBody ["profile"]
+--------------------------------------------------------------------------------
+-- TRANSACTIONS ----------------------------------------------------------------
+send :: forall eff. CAddress -> CAddress -> Coin -> Aff (ajax :: AJAX | eff) CTx
+send addrFrom addrTo amount = postR ["txs/payments", _address addrFrom, _address addrTo, show $ _coin amount]
 
-restoreWallet :: forall eff. CWalletInit -> Aff (ajax :: AJAX | eff) CWallet
-restoreWallet = postRBody ["restore_wallet"]
+sendExtended :: forall eff. CAddress -> CAddress -> Coin -> CCurrency -> String -> String -> Aff (ajax :: AJAX | eff) CTx
+sendExtended addrFrom addrTo amount curr title desc = postR ["txs/payments", _address addrFrom, _address addrTo, show $ _coin amount, showCCurrency curr, title, desc]
 
-nextUpdate :: forall eff. Aff (ajax :: AJAX | eff) CUpdateInfo
-nextUpdate = getR ["next_update"]
+updateTransaction :: forall eff. CAddress -> CTxId -> CTxMeta -> Aff (ajax :: AJAX | eff) Unit
+updateTransaction addr ctxId = postRBody ["txs/payments", _address addr, _ctxIdValue ctxId]
 
-applyUpdate :: forall eff. Aff (ajax :: AJAX | eff) Unit
-applyUpdate = postR ["apply_update"]
+getHistory :: forall eff. CAddress -> Int -> Int -> Aff (ajax :: AJAX | eff) (Tuple (Array CTx) Int)
+getHistory addr skip limit = getR ["txs/histories", _address addr, show skip, show limit]
 
-systemVersion :: forall eff. Aff (ajax :: AJAX | eff) SoftwareVersion
-systemVersion = getR ["system_version"]
+searchHistory :: forall eff. CAddress -> String -> Int -> Int -> Aff (ajax :: AJAX | eff) (Tuple (Array CTx) Int)
+searchHistory addr search skip limit = getR ["txs/histories", _address addr, search, show skip, show limit]
```

Comment:
https://github.com/input-output-hk/cardano-sl/pull/78/files#r104671937 

Please make these corrections in this file so I don't cp.